autoinstall/FreeBSD/13/installerconfig

```diff
@@ -78,6 +78,21 @@ done
 
 # Disable ISO repo and enable default repo
 rm -rf /usr/local/etc/pkg/repos/FreeBSD_install_cdrom.conf
+
+# Modify the offical repo to latest
+{% if guest_id.find('64Guest') == -1 %}
+mkdir -p /usr/local/etc/pkg/repos
+cat >/usr/local/etc/pkg/repos/FreeBSD.conf <<EOF
+FreeBSD: {
+    url: "pkg+http://pkg.FreeBSD.org/\${ABI}/latest",
+    mirror_type: "srv",
+    signature_type: "fingerprints",
+    fingerprints: "/usr/share/keys/pkg",
+    enabled: yes
+}
+EOF
+{% endif %}
```

Comment:
What if keep both of quarterly repo and latest repo?  And I think this change can be applied to both 32bit and 64bit FreeBSD. So that we can install same version of packages on 32bit and 64bit FreeBSD

Reply:
> What if keep both of quarterly repo and latest repo?
The Note in file /etc/pkg/FreeBSD.conf :
    To disable this repository. Instead of modifying or removing this file. create a /usr/local/etc/pkg/repos/FreeBSD.conf file.

> And I think this change can be applied to both 32bit and 64bit FreeBSD. So that we can install same version of packages on 32bit and 64bit FreeBSD

OK